example/src/App.css

```
@@ -1,225 +1,708 @@
-  color-scheme: light dark;
+  color: #142420;
+  background: #f4f7f5;
+  font-family:
+    Inter,
+    ui-sans-serif,
+    -apple-system,
+    BlinkMacSystemFont,
+    "Segoe UI",
+    sans-serif;
+  font-synthesis: none;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.14.1)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)